packages/graphql-typescript-definitions/src/cli.ts

```diff
@@ -2,21 +2,28 @@
 
 import * as yargs from 'yargs';
 import chalk from 'chalk';
+import {relative} from 'path';
 
 import {EnumFormat} from './types';
 import {Builder} from '.';
 
 const argv = yargs
   .usage('Usage: $0 <graphql-files> [options]')
+  .option('cwd', {
+    required: false,
+    normalize: true,
+    type: 'string',
+    describe: 'Working directory to use',
+  })
   .option('schema-path', {
-    required: true,
+    required: false,
     normalize: true,
     type: 'string',
     describe:
       'The path to the JSON file containing a schema instrospection query result',
   })
   .option('schema-types-path', {
-    required: true,
+    required: false,
```

Comment:
the cli supports one or multiple schemas through the `.graphqlconfig` or alternatively supports a single schema through these cli params (and no `.graphqlconfig`). The happy path is to always use the config, so that's why these were no longer required. Basically this is just for backwards compat.

Reply:
based on your other comment, i'll make this required again.

Reply:
Is this going to be required? I feel this is disconnected from the graphql config — that is for "where are my schema and documents", this is for "where do you want your types".